### PR TITLE
compile with -dip1000

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

**Comment by WalterBright** [comment]
Blocked by https://github.com/dlang/dmd/pull/6420

**Comment by andralex** [comment]
Auto-merge toggled on

**Comment by WalterBright** [comment]
Hmm, looks like I have a little more work to do here.

File: src/rt/config.d [hunk 1]
```diff
@@ -58,7 +58,7 @@ import core.stdc.string : strlen;
 
 extern extern(C) string[] rt_args() @nogc nothrow;
 
-alias rt_configCallBack = string delegate(string) @nogc nothrow;
+alias rt_configCallBack = string delegate(return scope string) @nogc nothrow;
```
Comment (ZombineDev) [inline src/rt/config.d]: You probably also need to update https://github.com/dlang/druntime/blob/v2.073.0-b1/src/gc/config.d#L43

**Comment by WalterBright** [comment]
Now blocked by https://github.com/dlang/dmd/pull/6429

**Pull request merged**